internal/db/job_tracker_test.go

```diff
@@ -30,6 +30,32 @@ func (s *dbSuite) TestJobTracker_CreateJob(c *qt.C) {
 	c.Assert(entry.Error, qt.Equals, "")
 }
 
+func (s *dbSuite) TestJobTracker_GetJobStopSignal(c *qt.C) {
+	ctx := c.Context()
+	err := s.Database.Migrate(ctx)
+	c.Assert(err, qt.IsNil)
+
+	_, err = s.Database.GetJobStopSignal(ctx, uuid.New())
```

Comment:
for completeness, can you also try it before migration?

Reply:
Yup